api/core/rag/rerank/weight_rerank.py

```diff
@@ -39,9 +39,8 @@ def run(
         unique_documents = []
         doc_id = set()
```

Comment:
How about change the name into `doc_id` to prevent ambiguous usage?

Reply:
sure, doc_ids is better